packages/gatsby/src/redux/actions/internal.ts

```diff
@@ -445,3 +452,99 @@ export const clearGatsbyImageSourceUrls =
       type: `CLEAR_GATSBY_IMAGE_SOURCE_URL`,
     }
   }
+
+// We add a counter to node.internal for fast comparisons/intersections
```

Comment:
the additions here were just moved from `public.js` file here to make it easier to reuse